xbmc/cores/paplayer/VideoPlayerCodec.cpp

```diff
@@ -510,7 +512,10 @@ CAEStreamInfo::DataType VideoPlayerCodec::GetPassthroughStreamType(AVCodecID cod
       break;
 
     case AV_CODEC_ID_DTS:
-      format.m_streamInfo.m_type = CAEStreamInfo::STREAM_TYPE_DTSHD;
+      if (profile == FF_PROFILE_DTS_HD_HRA)
+        format.m_streamInfo.m_type = CAEStreamInfo::STREAM_TYPE_DTSHD;
+      else
```

Comment:
same thing here, should we be explicit about `FF_PROFILE_DTS_HD_MA `?

Reply:
See above reply. If it's not DTS-HD-HR, we need to make sure to support 192 / 8 - and this is DTSHD_MA in kodi speach.